daily-challenges/2024-12/13/lcaloi.29.spec.ts

```
@@ -0,0 +1,92 @@
+    kiemTraTrangThai(): string {
+        if (!this.thoiGianBatDau || !this.thoiGianThuHoach) {
+            return 'Cây chưa được trồng'
+        }
+
+        if (this.trangThai === 'da thu hoach') {
+            return this.trangThai;
+        }
+
+        if (this.thoiGianBatDau.getTime() >= this.thoiGianThuHoach.getTime()) {
+            return this.trangThai = 'san sang thu hoach';
+        }
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)